src/importer/Program.cs

```diff
@@ -11,36 +11,35 @@ static void Main(string[] args)
+            var apiOption = app.Option("-u|--url|--api-url <url>", "URL of the ManageCourses API", CommandOptionType.SingleValue);
+            var apiKeyOption = app.Option("-k|--key|--api-key <key>", "Admin Key for the ManageCourses API", CommandOptionType.SingleValue);
+            
             app.HelpOption("-?|-h|--help");
             app.Execute(args);
 
-            var users = new CsvReader().ReadUsers(folderOption.Value());
-            var organisations = new CsvReader().ReadOrganisations(folderOption.Value());
-            var organisationInstitutions = new CsvReader().ReadOrganisationInstitutions(folderOption.Value(), organisations);
-            var organisationUsers = new CsvReader().ReadOrganisationUsers(folderOption.Value(), organisations, users);
+            // only used to avoid importing orphaned campuses
             var institutions = new XlsReader().ReadInstitutions(folderOption.Value());
+            
+            // data to import
             var campuses = new XlsReader().ReadCampuses(folderOption.Value(), institutions);
             var courses = new XlsReader().ReadCourses(folderOption.Value(), campuses);
             var subjects = new XlsReader().ReadSubjects(folderOption.Value());
             var courseSubjects = new XlsReader().ReadCourseSubjects(folderOption.Value(), courses, subjects);
             var courseNotes = new XlsReader().ReadCourseNotes(folderOption.Value());
             var noteTexts = new XlsReader().ReadNoteText(folderOption.Value());
 
-            var payload = new Payload
+            var payload = new UcasPayload
             {
                 Courses = new ObservableCollection<UcasCourse>(courses),
-                Institutions = new ObservableCollection<UcasInstitution>(institutions),
                 CourseSubjects = new ObservableCollection<UcasCourseSubject>(courseSubjects),
                 Subjects = new ObservableCollection<UcasSubject>(subjects),
                 Campuses = new ObservableCollection<UcasCampus>(campuses),
                 CourseNotes = new ObservableCollection<UcasCourseNote>(courseNotes),
-                NoteTexts = new ObservableCollection<UcasNoteText>(noteTexts),
-                Users = new ObservableCollection<McUser>(users),
-                Organisations = new ObservableCollection<McOrganisation>(organisations),
-                OrganisationInstitutions = new ObservableCollection<McOrganisationInstitution>(organisationInstitutions),
-                OrganisationUsers = new ObservableCollection<McOrganisationUser>(organisationUsers)
+                NoteTexts = new ObservableCollection<UcasNoteText>(noteTexts)
             };
-            new ManageApi().SendToManageCoursesApi(payload);
+            
+            var apiKey = apiKeyOption.HasValue() ? apiKeyOption.Value() : "";
+            new ManageApi(apiOption.Value(), apiKey).PostPayload(payload);
```

Comment:
variable names? apiOptions vs apiKey?

Reply:
If there is a point then new it. Otherwise pointless...

Reply:
it was hard to read when inlined...